utils.js

```diff
@@ -16,7 +23,7 @@ export async function DiscordRequest(endpoint, options) {
 
     if (!res.ok) {
         const data = await res.json();
-        console.log(res.status);
+        log.error(`Unable to send discord request: Error ${res.status}`);
```

Comment:
Noice. We should clean up the old unlabeled logs in another PR (or add labels to them) to reduce noise and to make finding them in code easier